alvr/server_core/src/tracking/vmc.rs

```diff
@@ -28,6 +28,8 @@ static DEVICE_MOTIONS_VMC_MAP: Lazy<HashMap<u64, &'static str>> = Lazy::new(|| {
     ])
 });
 
+// These values were measured, so having them with full precision makes sense
+#[expect(clippy::excessive_precision)]
```

Comment:
The problematic value is `-0.7071100` at BODY_LEFT_FOOT_ID. original it has 5 decimal digita like the pther values, and the extra 2 zeros were added because of the lint regarding "emulating PI/2". so i'd say it's better to remove the extra 2 zeros and instead expect() the lint about PI

Reply:
Fair enough